caliban/history/types.py

```diff
@@ -160,7 +151,7 @@ def get_or_create(
       session: Session,
       spec: Dict[str, Any],
       user: Optional[str] = None,
-  ) -> ContainerSpec:
+  ) -> "ContainerSpec":
```

Comment:
this, I guess, is a loss from 3.7-> 3.6 (annotations)...seems like a reasonable price to pay for avoiding the configuration headaches

Reply:
Yup, that's the only loss, really. We get our f-strings, and we don't get forward declarations...